chainstate/test-suite/src/tests/fungible_tokens.rs

```diff
@@ -753,7 +753,8 @@ fn transfer_split_and_combine_tokens(#[case] seed: Seed) {
         let token_min_issuance_fee = tf.chainstate.get_chain_config().token_min_issuance_fee();
 
         // Issue a new token
-        let genesis_outpoint_id = TestBlockInfo::from_genesis(&tf.genesis()).txns[0].0.clone();
+        let genesis_outpoint_id =
```

Comment:
Agree that this looks ugly. But I think the intention here was to get `OutPointSourceId` and it can be simply rewritten as:
```rust 
let genesis_outpoint_id = OutPointSourceId::BlockReward(tf.genesis().get_id().into());
```
No need to collect all outputs to create an id.

Reply:
Updated. Much better now, thanks!